plugins/woocommerce/tests/php/src/Internal/Admin/Settings/UtilsTest.php

```diff
@@ -1463,4 +1463,113 @@ public function data_provider_order_map_normalize(): array {
 			),
 		);
 	}
+
+	/**
+	 * Test getting testing plugin slug suffixes list
+	 */
+	public function test_get_testing_plugin_slug_suffixes() {
```

Comment:
I am not sure this test is adding value since the method under test simply returns a hard-coded list of suffixes.

Reply:
Yeah. I agree. I just wanted to have test coverage for it :)